Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN set -xeu && \
         && \
     pip install --no-cache-dir \
         gevent==1.2.2 `# needed for gunicorn -k gevent` \
+        psycopg2-binary `# for superset own database` \
```

Comment:
does it pick it up automatically?
Is this visible for user or just for superset metadata?

Reply:
I don't understand? It won't be visible to users unless you define a data source reading from some postgres database. 
(Although, FWIW, the superset's metadata database is also available as data source in superset UI)